src/commands/auth/logout.js

```diff
@@ -20,11 +20,15 @@ class LogoutCommand extends ImsBaseCommand {
     const { invalidateToken, context } = require('@adobe/aio-lib-ims')
     const { CLI } = require('@adobe/aio-lib-ims/src/context')
     const current = await context.getCurrent()
-
     flags.ctx = flags.ctx || (current || CLI)
-
     try {
-      await invalidateToken(flags.ctx, flags.force)
+      const accessToken = await context.get(`${flags.ctx}.access_token`)
+      if (!accessToken.data) {
```

Comment:
Is there a better way to check if access_token exists ?

Reply:
not that I can see